content/guides/author-libraries.md

```diff
@@ -34,56 +77,70 @@ export function wordToNum(word) {
 };
 ```
 
-The usage spec for the library will be as follows.
+The usage specification for the library use will be as follows:
 
 ```javascript
-import * as webpackNumbers from 'webpack-numbers';
-
+import * as webpackNumbers from 'webpack-numbers';//ES2015 module import
+var webpackNumbers = require('webpack-numbers');// CommonJS module require
 ...
-webpackNumbers.wordToNum('Two') // output is 2
+webpackNumbers.wordToNum('Two') //ES2015 and CommonJS module use
 ...
+//AMD module require
+require(['webpackNumbers'], function ( webpackNumbers) {
+    ...
+    webpackNumbers.wordToNum('Two')//AMD module use
+    ...
+});
 
-// CommonJS modules
-
-var webpackNumbers = require('webpack-numbers');
-
-...
-webpackNumbers.numToWord(3); // output is Three
-...
 ```
 
-```html
-// Or as a script tag
+The consumer also can use the library loading it with the script tag:
 
+```html
 <html>
 ...
 <script src="https://unpkg.com/webpack-numbers"></script>
 <script>
     ...
-    /* webpackNumbers is available as a global variable */
-    webpackNumbers.wordToNum('Five') //output is 5
+    /* Global variable */
+    webpackNumbers.wordToNum('Five') 
+    /* Property in the window object */
+    window.webpackNumbers.wordToNum('Five') 
+    //
     ...
 </script>
 </html>
 ```
 
+The configurations also can expose the library in the next ways:
+
+- Property in the global object, for node.
+- Property in the this object.
+
+
 For full library configuration and code please refer to [webpack-library-example](https://github.com/kalcifer/webpack-library-example)
 
 
 ## Configure webpack
 
-Now the agenda is to bundle this library
+Now the agenda is to bundle this library achieving the next goals:
+
+- Without bundling `lodash`, but requiring it to be loaded by the consumer using `externals`.
+- Setting the library name as `webpack-numbers`.
+- Exposing the library as a variable called `webpackNumbers`.
+- Being able to access the library inside Node.js.s
+
+Also, the consumer will be able to access` the library the `next ways:
```

Comment:
This sentence has a stray code block and a grammar issue.

How about: "Also, the consumer will be able to access the library in the following ways:"